filtersrv/src/main/java/org/apache/rocketmq/filtersrv/FiltersrvStartup.java

```diff
@@ -139,7 +139,7 @@ public static FiltersrvController createController(String[] args) {
 
             Runtime.getRuntime().addShutdownHook(new ShutdownHookThread(log, new Callable() {
                 @Override
-                public Object call() throws Exception {
+                public Void call() throws Exception {
                     controller.shutdown();
                     return null;
```

Comment:
@WillemJiang, Are you sure about no need `return null`? I have run a `mvn package` on local, and you can see, `return statement` is required.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project rocketmq-namesrv: Compilation failure
[ERROR] /Users/wusheng/Documents/code/github/incubator-rocketmq-fork/namesrv/src/main/java/org/apache/rocketmq/namesrv/NamesrvStartup.java:[128,17] 缺少返回语句
[ERROR] -> [Help 1]
```